static/css/chara_manager.css

```
@@ -473,295 +482,298 @@ padding: 0 15px;
-max-width: fit-content;
-opacity: 0.7;
-border-radius: 4px;
+    color: #aaa;
+    background: transparent;
+    font-style: italic;
+    font-weight: normal;
+    padding: 4px 8px;
+    margin-bottom: 0;
+    font-size: 0.75rem;
+    font-family: 'Segoe UI', Arial, sans-serif;
+    overflow-wrap: break-word;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "overflow-wrap" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)